src/com/cap/omts/dao/AdminDAOImpl.java

```diff
@@ -72,7 +69,7 @@ public int addTheatre(Theatre theatre) throws OMTSException {
 					throw new OMTSException("problem while closing Database");
 				}
 			}
-		return insertedRows;
+		return rows;
 	}
 	
 	

```

Comment:
please change the insertedRows variable to an appropriate name. I suggest you to name it as isInserted and initialise it.